common/supplemental/supplementalData.xml

```diff
@@ -1475,12 +1475,12 @@ XXX Code for transations where no currency is involved
 		<language type="cop" scripts="Arab Copt Grek" territories="EG" alt="secondary"/>
 		<language type="cps" scripts="Latn"/>
 		<language type="cr" scripts="Cans Latn"/>
-		<language type="cr" territories="CA" alt="secondary"/>
 		<language type="crg" scripts="Latn"/>
 		<language type="crh" scripts="Cyrl"/>
 		<language type="crj" scripts="Cans"/>
 		<language type="crj" scripts="Latn" alt="secondary"/>
 		<language type="crk" scripts="Cans"/>
+		<language type="crk" territories="CA" alt="secondary"/>
```

Comment:
This change happened because Woods Cree is not an official language of a region of Canada.

Rather, Plains Cree is. Cree is only official in the Northern Territories (NT). Unfortunately, [the NT law](https://web.archive.org/web/20090324202430/http://www.justice.gov.nt.ca/PDF/ACTS/Official_Languages.pdf) does not specify which Cree variation. We can deduce the variation because [the only Cree language present in NT is Plains Cree [crk]](https://en.wikipedia.org/wiki/Cree_language#Dialect_criteria) so I infer that is the correct match.

Reply:
Sounds good